examples/flutter_gallery/ios/fastlane/Fastfile

```diff
@@ -46,14 +46,15 @@ platform :ios do
 
     puts 'Retrieving signing certificates and profiles...'
     # Stop fastlane from echoing back PUBLISHING_MATCH_CERTIFICATE_REPO var.
-    suppress_output {
+    # suppress_output {
       # Retrieves all the necessary certs and provisioning profiles.
       sync_code_signing(
-        git_url: ENV['PUBLISHING_MATCH_CERTIFICATE_REPO'],
+        git_url:  "https://x-access-token:#{ENV['CIRRUS_REPO_CLONE_TOKEN']}@#{ENV['PUBLISHING_MATCH_CERTIFICATE_REPO']}" ,
```

Comment:
hey @fkorotkov, sorry to bug you again. We need to make use of git in this ruby script again with the same user session as used to clone the repo itself. This env doesn't seem to be populated. Is this the right env to look at or should we be using git some other way to authenticate?

Reply:
@xster I think this part of documentation can help to answer your question: https://cirrus-ci.org/guide/writing-tasks/#custom-clone-command

Reply:
That's where I got this env variable name from. Though it seems unpopulated in this PR run

Reply:
It should be. By default `clone` is performed via `go-git` and not `git` executable. Not sure if it interferes with your script.

Reply:
Thanks. I tried printing out various env variables listed in https://cirrus-ci.org/guide/writing-tasks/#environment-variables but only CIRRUS_REPO_CLONE_TOKEN was specifically not populated. 

Doesn't really matter too much. We'll just use our own value via encrypted variables instead. Thanks!